### ISSUE TITLE
Fix the login page UI and Redesign it

### ISSUE THREAD

**Issue by AyishikD** [AyishikD/Photogram#11]
### Describe the feature

Fix the login page UI with buttons for different sign in options according to the given ones

### Would you like to work on this issue?

None

**Comment by dipa-ghosh11** [comment]
Yes i am interested in this issue. I will create sign in page and register section (if required) with button. Please assign me for this issue @AyishikD

**Comment by AyishikD** [comment]
> Yes i am interested in this issue. I will create sign in page and register section (if required) with button. Please assign me for this issue @AyishikD

Best of luck @dipa-ghosh11 you have 3 days to fix it

**Comment by dipa-ghosh11** [comment]
I have have sent pull request please merge it and suggest me if any changes required. Do i need to create sign up section? please check @AyishikD

**Comment by AyishikD** [comment]
Work progress @dipa-ghosh11 ?

**Comment by IndraniSom** [comment]
Please assign me in this issue under Apertre-24

**Comment by AyishikD** [comment]
Best of luck @IndraniSom
